src/rules/flake8_annotations/rules.rs

```diff
@@ -332,6 +371,15 @@ pub fn definition(checker: &mut Checker, definition: &Definition, visibility: &V
                     }
                 }
             }
+            // If settings say so, don't report any of the
+            // diagnostics gathered here if there were no type annotations at all.
+            if checker.settings.flake8_annotations.ignore_fully_untyped
```

Comment:
I was hoping we could just have this setting, and not `allow_untyped_return_if_untyped` :joy: What's an example of where `allow_untyped_return_if_untyped` would be useful but you'd want `ignore_fully_untyped` to be kept false?

Reply:
Ah yeah, you're right, `ignore_fully_untyped` alone should be enough.